code/__defines/gamemode.dm

```diff
@@ -12,19 +12,6 @@
 #define BE_SYNTH "BE_SYNTH"
 #define BE_PAI   "BE_PAI"
 
-// Antagonist datum flags.
```

Comment:
doesnt the storyteller use any of these? It probably should.

Reply:
But it doesn't.